Validation/MajorRoads/UBA-DE_HVS_to_END_DF7_10_Roads.halex.alignment.xml

```diff
@@ -510,7 +510,13 @@ withTransformationContext {	&#13;
 		publicConsultation_commentsReceived( _snippets.helperFunctions.classifyJaNein(c.Öffentlichkeitsbeteiligung.kommentareEingegangen.value()) )&#13;
 		publicConsultation_commentsIncludedInNAP( _snippets.helperFunctions.classifyJaNein(c.Öffentlichkeitsbeteiligung.kommentareAufgenommen.value()) )&#13;
 		publicConsultation_NAPReviewed( _snippets.helperFunctions.classifyJaNein(c.Öffentlichkeitsbeteiligung.lapUeberarbeitet.value()) )&#13;
+		&#13;
+		if (c.Öffentlichkeitsbeteiligung.lapUeberarbeitet.value()  == 'Nein' &amp;&amp; !c.Öffentlichkeitsbeteiligung.lapUeberarbeitetKommentar.value()) {&#13;
+			publicConsultation_reviewExplanation( 'Eine Überarbeitung des Lärmaktionsplans nach der Öffentlichkeitsbeteiligung war nicht erforderlich.' )&#13;
+		}&#13;
+		else {&#13;
 		publicConsultation_reviewExplanation( c.Öffentlichkeitsbeteiligung.lapUeberarbeitetKommentar.value() )&#13;
+		}&#13;
```

Comment:
Code style: wrong indentation of line 518

Reply:
Don't what was the issue here, because for me it looked correct in the groovy editor in hale studio.
But I removed and added the indentation with an editor and now it should be correct.